db/repos/accounts.js

```diff
@@ -40,8 +40,8 @@ const normalFields = [
 	{ name: 'fees', cast: 'bigint', def: '0', skip: ifNotExists },
 	{ name: 'rewards', cast: 'bigint', def: '0', skip: ifNotExists },
 	{ name: 'vote', cast: 'bigint', def: '0', skip: ifNotExists },
-	{ name: 'producedBlocks', cast: 'bigint', def: '0', skip: ifNotExists },
-	{ name: 'missedBlocks', cast: 'bigint', def: '0', skip: ifNotExists },
+	{ name: 'producedBlocks', def: 0, skip: ifNotExists },
```

Comment:
(Not related to PR but) aren't `def: 0` and `skip: ifNotExists` conflicting? Meaning if default is 0 then it will always exist and never be skipped?

Reply:
@SargeKhan `skip` is used by `update` and `sets` and ignored by `inserts` and `values`. So the `def` covers the use case of inserting records.